src/radembedded.c

```diff
@@ -74,14 +83,30 @@ main (int argc, char **argv)
 	 * entry to specify the location of the 'servers' file which stores the secrets to
 	 * be used.
 	 */
-	if (rc_add_config(rh, "authserver", "localhost::testing123", "config", 0) != 0)
+	/* If the IP Address is provided via Command-line, take it for processing. Else,
+	 * use localhost as default.
+	 */
+	if(argc == 2)
+	{
+		strncpy(auth_server_ip, argv[1], sizeof(auth_server_ip));
```

Comment:
Let's not get into trouble by using strncpy or strncat. They may not null terminate the string. snprintf() may be a better choice here.

Reply:
Hi @nmav 

Thanks for the review. Addressed your comments, used snprintf instead of string APIs. 
Kindly review the changes and let me know your views.